topics/single-cell/tutorials/pseudobulk-analysis/tutorial.md

```diff
@@ -0,0 +1,523 @@
+- Which genes are most significant?
+- How does the fold change correlate with the P-values?
+- Are there any unexpected patterns?
+
+> <question-title> Reflect on the Volcano Plot </question-title>
+>
+> 1. What is the significance of genes located at the extremes of the plot (e.g., high LogFC and low P-value)? 
+> 2. How many genes meet the significance threshold in this analysis?
+>
+> > <solution-title> Solution </solution-title>
+> >
+> > 1. Genes at the extremes of the Volcano plot are highly significant and exhibit strong differential expression. Consider exploring the biological significance of these genes to gain deeper insights into their roles.
+> > 2. In our current analysis, we identified only one downregulated gene, MTND1P23, with a logFC of -3.1956, in normal conditions compared to COVID-19 patients.
+> >
+> {: .solution}
+>
+{: .question}
+
+# Subsetting Samples from the Original AnnData Object
+
+In our previous analysis, we found that only one gene, **MTND1P23**, was identified as downregulated in our contrast. This result was obtained by "bulking" **all** cell types from our dataset and performing a differential expression analysis comparing normal vs. COVID-19 samples.
+
+Now, what if we refine our approach? For instance, instead of analysing all cell types together, what happens if we focus on a specific cluster, such as **T cells**, and perform the same comparison: normal vs. COVID-19? Would the results remain the same, or would this approach reveal additional insights?
+
+## Extracting observations of interest as AnnData object 
+
+> <hands-on-title> Use Manipulate AnnData Tools to extract observations </hands-on-title>
+>
+> 1. Use the {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.10.9+galaxy0) %} tool with the following parameters:
+>    - *"Annotated data matrix"*: `AnnData for Pseudobulk` (Your preprocessed, analysed, and annotated AnnData object)  
+>    - *"Function to manipulate the object"*: `Filter observations or variables`  
+>    - *"What to filter?"*: `Observations (obs)` (For filtering cells, select Observations (obs))  
+>    - *"Type of filtering?"*: `By key (column) values`  
+>    - *"Key to filter"*: `cell_type` (the label that identifies cell annotations in our AnnData)  
+>    - *"Type of value to filter"*: `Text`  
+>    - *"Filter"*: `equal to`  
+>    - *"Value"*: `T cell` (the name of the cluster of interest for subset analysis)
+{: .hands_on}
+>
+>  After using the {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.10.9+galaxy0) %} tool to subset the cell type of interest, go back to the top of this tutorial to the hands-on **Pseudobulk with Decoupler** step, and you may perform once again the same steps in this smaller AnnData object that now should only include your T cells. Results from this analysis will correspond to differential expression between conditions only for T cells. 
+> 
```

Comment:
Since this is outside of the box, can remove the `>` characters 

```suggestion

After using the {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.10.9+galaxy0) %} tool to subset the cell type of interest, go back to the top of this tutorial to the hands-on **Pseudobulk with Decoupler** step, and you may perform once again the same steps in this smaller AnnData object that now should only include your T cells. Results from this analysis will correspond to differential expression between conditions only for T cells. 

```